### ISSUE TITLE
Screen reader in scan mode targeting invisible elements

### ISSUE THREAD

**Issue by gikoifma** [highcharts/highcharts#16830]
#### Expected behaviour
When scanning charts and reading chart related information in "highcharts-screen-reader-region-before", the screen reader will show the focus on the chart itself.

#### Actual behaviour
Screen readers focus the div mentioned above, and as a result users see the invisible element marked.
This is causing difficulties to the users to understand which chart is the screen reader talking about, especially when there are several charts lined up.

#### Live demo with steps to reproduce
Happens also in the demos in your website. Attaching an image that shows that blue rectangle on the top left(Narrator on Chrome):
![image](https://user-images.githubusercontent.com/9548935/147556215-4718bf1d-4f94-4b80-be18-76495f23d000.png)

#### Product version
9.3.1

#### Affected browser(s)
Edge, Chrome

**Comment by raf18seb** [comment]
Hi @Painese, thank you for creating the ticket.
We'll investigate further and provide more info at the beginning of January. Sorry for the inconvenience.

**Comment by KacperMadej** [comment]
@Painese I am unable to recreate the problem. Please provide more details about the software you are using - names and versions. Steps needed to recreate the problem would also be nice to ensure we are looking at the same problem.

**Comment by Painese** [comment]
Sure.
First versions:
Narrator + Chrome versin 96.0.4664.110.
This repro with NVDA also and with Edge.

Lets take the following URL for the repro:
https://www.highcharts.com/demo/line-log-axis

I start with the screen reader focused on the left arrow:
![image](https://user-images.githubusercontent.com/9548935/147829575-78a6bb55-ceb2-4a0e-951d-92b018a942ec.png)

Then I click on the "down arrow" key. It moves me to the screen reader section of the chart, reading out loud: "Logarithmic axis demo".
After it finishes reading this line, the screen reader marks the small empty square to the left.
![image](https://user-images.githubusercontent.com/9548935/147829614-370c12f8-0ce5-4ff6-96d7-b5bd4af85bde.png)

**Comment by oysteinmoseng** [comment]
Hi @Painese, could you elaborate a bit how you would want this to work ideally?

This behavior happens because the off-screen text is styled and positioned to be invisible and not interfere with other page elements. By default, the chart title is read as part of the text, which should help with identifying which chart is being explored, but the most accessible solution is to make the text visible. Let me know if making the text visible is an option for you, and I can provide an example of how to achieve this.

One enhancement that is already planned is to make the chart title text itself accessible, which means that when screen readers read out "Logarithmic axis demo" in the demo above, the title text would be outlined with the blue square.

**Comment by Painese** [comment]
"One enhancement that is already planned is to make the chart title text itself accessible, which means that when screen readers read out "Logarithmic axis demo" in the demo above, the title text would be outlined with the blue square."

@oysteinmoseng That enhancement should be enough in order resolve my issue :)
Thank you for looking into it!
